Meadowlark-js/package.json

```diff
@@ -52,12 +51,13 @@
     "test:e2e": "cross-env NODE_OPTIONS=--max-old-space-size=4096 LOG_LEVEL=warn npx jest e2e --config ./jest.e2e-config.js --runInBand",
     "test:system:ci": "npm run test:system:mongodb  -- --ci --config ./jest.ci-config.js && npm run test:system:postgresql  -- --ci --config ./jest.ci-config.js",
     "test:unit:coverage": "rimraf coverage/ && npm run test:unit -- --collectCoverage",
-    "prepare": "cd .. && husky install",
-    "start:local": "lerna run start:local"
-  },
-  "husky": {
-    "hooks": {
-      "pre-push": "npm run test:lint"
-    }
```

Comment:
As discussed in Slack, removed husky above.

Below, note all of the new docker commands. I wish there were a way to add comments right into the package.json. I will look for a place to describe these commands in a markdown file.

For QA purposes:

* Run `docker:build` to create a docker image
* Run `docker:up` to startup the Compose environment
* Run `docker:down` to shutdown that environment.

The other commands are for low-level testing on the individual image, not using the Compose environment.

Reply:
Given these changes, could we remove the prepare script and husky from the dependencies?

Reply:
They _are_ removed above... but they also appear in meadowlark-fastify, I now see. So I'll remove those as well.